xcape/test/test_core.py

```diff
@@ -37,13 +37,17 @@ def p_t_td_surface(nx=10, ny=5):
 @pytest.mark.parametrize('use_dask', [False, True])
 @pytest.mark.parametrize('sourcein,n_returns',
                          [('surface', 2), ('most-unstable', 4)])
-def test_calc_cape_shape_3d(p_t_td_3d, p_t_td_surface, sourcein, n_returns, use_dask):
+@pytest.mark.parametrize('vertical_levin', ['sigma', 'pressure'])
+def test_calc_cape_shape_3d(p_t_td_3d, p_t_td_surface, sourcein, n_returns, use_dask,vertical_levin):
     p, t, td = p_t_td_3d
     ps, ts, tds = p_t_td_surface
-    args = (p, t, td, ps, ts, tds)
+    if vertical_levin=='sigma':
+        args = (p, t, td, ps, ts, tds)
+    elif vertical_levin =='pressure':        
+        args = (p, t, td, ps, ts, tds,ps*0+1)
     if use_dask:
```

Comment:
ok now that i see this, I should go back to cape_dummy and add pres_lev_pos. I will do it next.